LFPAnalysis/lfp_preprocess_utils.py

```diff
@@ -798,22 +798,43 @@ def make_mne(load_path=None, elec_path=None, format='edf', site='MSSM', resample
         elif sync_type == 'ttl': 
             print('TTL  used - no need to split out a separate sync channel. Check the .nev file with the neural data.')
 
-
-        # The electrode names read out of the edf file do not always match those 
-        # in the pdf (used for localization). This could be error on the side of the tech who input the labels, 
-        # or on the side of MNE reading the labels in. Usually there's a mixup between lowercase 'l' and capital 'I'.
         
-        # Sometimes, there's electrodes on the pdf that are NOT in the MNE data structure... let's identify those as well. 
-        new_mne_names, _, _ = match_elec_names(mne_data.ch_names, elec_data.label)
-        # Rename the mne data according to the localization data
-        new_name_dict = {x:y for (x,y) in zip(mne_data.ch_names, new_mne_names)}
-        mne_data.rename_channels(new_name_dict)
+        if site=='MSSM':
 
-        if not seeg_names:
-            seeg_names = [i for i in mne_data.ch_names if (((i.startswith('l')) | (i.startswith('r'))) & (i!='research'))]
-        sEEG_mapping_dict = {f'{x}':'seeg' for x in seeg_names}
+            # The electrode names read out of the edf file do not always match those 
+            # in the pdf (used for localization). This could be error on the side of the tech who input the labels, 
+            # or on the side of MNE reading the labels in. Usually there's a mixup between lowercase 'l' and capital 'I'.
+            
+            # Sometimes, there's electrodes on the pdf that are NOT in the MNE data structure... let's identify those as well. 
+            new_mne_names, _, _ = match_elec_names(mne_data.ch_names, elec_data.label)
+            # Rename the mne data according to the localization data
+            new_name_dict = {x:y for (x,y) in zip(mne_data.ch_names, new_mne_names)}
+            mne_data.rename_channels(new_name_dict)
+
+            if not seeg_names:
+                seeg_names = [i for i in mne_data.ch_names if (((i.startswith('l')) | (i.startswith('r'))) & (i!='research'))]
+            sEEG_mapping_dict = {f'{x}':'seeg' for x in seeg_names}
+
+            mne_data.set_channel_types(sEEG_mapping_dict)
+            
+        elif site=='UCD':
+            
+            #UCD sometimes has messy naming conventions - before checking if mne names match recon labels, clean mne names 
+            mne_names = davis_utils.UCD_check_edf_names(mne_data.ch_names) #output is lowercase
+            # calling match_elec_names to make sure channel names in anat recon and mne_data are the same
+            new_mne_names, unmatched_names, unmatched_seeg = match_elec_names(mne_names, elec_data.label)
```

Comment:
lines 825-833 onwards seem very similar to lines 809-818. Can we condense the code a bit so that the 'if' condition really only runs MSSM or Davis-specific stuff, while we run other code 1x after the 'if' condition?